src/Pages/Shop/ShopDetails.js

```diff
@@ -1,7 +1,163 @@
 import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import ShopDetailsData from "./ShopDetailsData";
+import Option from "./Option";
+import TempText from "./TempText";
+import "./ShopDetails.scss";
 
 export default class ShopDetails extends Component {
+  constructor() {
+    super();
+    this.state = {
+      detailData: {},
+      imgClicked: "",
+      zoomed: false,
+      sidePanel: false,
+      isSelectActive: false,
+    };
+  }
+  componentDidMount() {
+    fetch("http://10.58.3.133:8000/product/detail/4")
```

Comment:
```suggestion
    fetch(`{API}/product/detail/4`)
```
루트 폴더 아래 config.js 만들어 주시고 거기에 api 모아서 관리해주세요